frameworks/PHP/swoole/database.php

```diff
@@ -0,0 +1,185 @@
+    {
+        $db->execute([mt_rand(1, 10000)]);
+        return json_encode($db->fetch(PDO::FETCH_ASSOC), JSON_NUMERIC_CHECK);
+    }
+
+    public static function fortunes(PDOStatement|PDOStatementProxy $fortune): string
+    {
+        $fortune->execute();
+        $results    = $fortune->fetchAll(PDO::FETCH_KEY_PAIR);
+        $results[0] = 'Additional fortune added at request time.';
+        asort($results);
+
+        $html = '';
+        foreach ($results as $id => $message) {
+            $message = htmlspecialchars($message, ENT_QUOTES, 'UTF-8');
+            $html    .= "<tr><td>$id</td><td>$message</td></tr>";
+        }
+
+        return "<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>$html</table></body></html>";
+    }
+
+    public static function query(PDOStatement|PDOStatementProxy $query, int $queries): string
+    {
+        $query_count = 1;
+        if ($queries > 1) {
+            $query_count = min($queries, 500);
+        }
+
+        $results = [];
+        while ($query_count--) {
+            $query->execute([mt_rand(1, 10000)]);
+            $results[] = $query->fetch(PDO::FETCH_ASSOC);
+        }
+
+        return json_encode($results, JSON_NUMERIC_CHECK);
+    }
+
+    public static function updates(PDOStatement|PDOStatementProxy $random, PDOStatement|PDOStatementProxy $update, int $queries): string
+    {
+        $query_count = 1;
+        if ($queries > 1) {
+            $query_count = min($queries, 500);
+        }
+
+        $results = [];
+        while ($query_count--) {
+            $id = mt_rand(1, 10000);
+            $random->execute([$id]);
+            $item = $random->fetch(PDO::FETCH_ASSOC);
+            $update->execute([$item['randomNumber'] = mt_rand(1, 10000), $id]);
+
+            $results[] = $item;
+        }
+
+        return json_encode($results, JSON_NUMERIC_CHECK);
+    }
+}
+
+class Connection
+{
+    private static PDOStatement $db;
+    private static PDOStatement $fortune;
+    private static PDOStatement $random;
+    private static PDOStatement $update;
+    private static PDOStatement $query;
+
+    public static function init(string $driver): void
+    {
+        $pdo = new PDO(
+            "$driver:host=tfb-database;dbname=hello_world",
+            "benchmarkdbuser",
+            "benchmarkdbpass",
+            [
+                PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+                PDO::ATTR_ERRMODE            => PDO::ERRMODE_EXCEPTION,
+                PDO::ATTR_EMULATE_PREPARES   => false
+            ]
+        );
+
+        self::$db      = $pdo->prepare(Operation::WORLD_SELECT_SQL);
+        self::$fortune = $pdo->prepare(Operation::FORTUNE_SQL);
+        self::$random  = $pdo->prepare(Operation::WORLD_SELECT_SQL);
+        self::$update  = $pdo->prepare(Operation::WORLD_UPDATE_SQL);
+        self::$query   = $pdo->prepare(Operation::WORLD_SELECT_SQL);
```

Comment:
I think that here you can also simplify it, without repeating the same SQL to create the prepared statement.